capstone/src/main/webapp/index.jsp

```diff
@@ -1,5 +1,14 @@
+<!DOCTYPE html> 
 <html>
-<body>
-<h2>Hello World!</h2>
-</body>
+  <head>
+  <meta charset="utf-8">
+  </head>
+  <body>
+    <h2>Welcome to UniViz!</h2>
+      <form action="/" method="POST" autocomplete="off">
```

Comment:
nit - consider leaving the action and method off of this form until they're needed.

Reply:
addressed!